Userland/Libraries/LibC/glob.cpp

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright (c) 2021, the SerenityOS developers.
```

Comment:
It's 2022!

Reply:
This commit has been sitting around (basically unchanged) since May of 2021, so I was unsure whether I should change it (even if just for sake of nostalgia). :^)

Reply:
wow! alright, let's keep it :)